Improve docs metadata (#405)
o1-labs/docs2: docs/zkapps/how-to-test-a-zkapp.mdx

---
title: How to Test a zkApp
hide_title: true

> description: This guide provides a detailed walkthrough on how to write and run tests for your zkApp using the Jest JavaScript testing framework. It includes creating tests for your smart contract, running tests, creating a local blockchain, deploying a contract locally, and writing integration tests.

Reviewer: ```suggestion
description: Guided steps to write and run tests for your zkApp using the Jest JavaScript testing framework, create and run tests, create a local blockchain, deploy a contract locally, and write integration tests.
```

Reviewer: thinking about how we search, let's use the verb "create" instead of the gerund "creating"

keywords:
  - zkApp

>   - Testing

Reviewer: ```suggestion
  - test a zkapp
```
